src/backend/renderer/element/surface.rs

```diff
@@ -235,6 +235,7 @@ pub fn render_elements_from_surface_tree<R, E>(
     location: impl Into<Point<i32, Physical>>,
     scale: impl Into<Scale<f64>>,
     alpha: f32,
+    kind: Option<Kind>,
```

Comment:
Why Option<> here?

Reply:
The idea behind making this an `Option` was in preparation of `wp_content_type`. On `None` it could map the content type defined by the client to a `Kind`. So providing `Some` would override the client defined content type.
But that might not be desired. Not sure what the best semantics are for this parameter, we could make it a fallback I guess. So if the surface does not specify a content type it would fallback to the provided `Kind`.

Reply:
@YaLTeR I changed it to not use an `Option` for now. When integrating `wp_content_type` we can decide how to handle that. We could still test for `Kind::Unspecified` and in this case use whatever content type the client defined for the surface if any.